core/services/pipeline/common_eth.go

```diff
@@ -204,6 +204,11 @@ func convertToETHABIBytes(destType reflect.Type, srcVal reflect.Value, length in
 		} else if srcVal.Type().Elem().Kind() != reflect.Uint8 {
 			return nil, errors.Wrapf(ErrBadInput, "cannot convert %v to %v", srcVal.Type(), destType)
 		}
```

Comment:
Isn't this an `else if` ?

Reply:
hm guess it doesnt matter since the error cases return, seems fine to keep the happy/error paths separate